src/HTTP/AxiosDispatcher.ts

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+import { Dispatcher } from '../api';
+
+export class AxiosDispatcher extends Dispatcher {
+  dispatch<T, R>(api: string, request: T): Promise<R> {
+    return axios.post(this.baseUrl + api, request)
+      .then(response => response.data)
+      .catch(error => Promise.reject(error.response.data));
```

Comment:
are you sure that the structure for error object will be always the same?

Reply:
That's right: response is optional in AxosError